shared_sql_utils/src/dsn.rs

```diff
@@ -38,12 +39,12 @@ pub struct DSNOpts {
     pub uri: String,
     pub user: String,
     pub server: String,
-    // SQL-1281
-    // pub logpath: String,
     pub driver_name: String,
+    pub log_level: String,
 }
 
 impl DSNOpts {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
We could avoid this with a specialized `struct` for the args, that way when we invoke this `new()` constructor, the arguments will all be clearly named/ordered.